api/gateway.proto

```diff
@@ -285,3 +290,18 @@ message GatewayConnectionStats {
   // Statistics for each sub band.
   repeated SubBand sub_bands = 10;
 }
+
+// The plain text value of the gateway secret.
+message GatewaySecretPlainText{
+  string value = 1;
```

Comment:
Encrypted data doesn’t have to be a string

Ideally it would be a marshaled proto that is also defined here, like `BasicStationLNSKey`, to make it fully typed.

Otherwise, consider `google.protobuf.Struct`.